src/api/libcellml/types.h

```diff
@@ -16,20 +16,24 @@ limitations under the License.
 
 #pragma once
 
+#include <any>
+#include <map>
 #include <memory>
 
+#include "libcellml/enums.h"
+
 namespace libcellml {
 
 // General classes.
 class Annotator; /**< Forward declaration of Annotator class. */
-using AnnotatorPtr = std::shared_ptr<Annotator>; /**< Type definition for shared annotator pointer. */
+using AnnotatorPtr = std::shared_ptr<Annotator>; /**< Type definition for @c std::shared Annotator pointer some othe text. */
 class Generator; /**< Forward declaration of Generator class. */
 using GeneratorPtr = std::shared_ptr<Generator>; /**< Type definition for shared generator pointer. */
 class GeneratorProfile; /**< Forward declaration of GeneratorProfile class. */
 using GeneratorProfilePtr = std::shared_ptr<GeneratorProfile>; /**< Type definition for shared generator variable pointer. */
 class GeneratorVariable; /**< Forward declaration of GeneratorVariable class. */
 using GeneratorVariablePtr = std::shared_ptr<GeneratorVariable>; /**< Type definition for shared generator variable pointer. */
-class Importer; /**< Forward declaration of Importer class. */
+class Importer; /*!< Forward declaration of Importer class. */
```

Comment:
Why the ! ?

Reply:
Again playing with Doxygen documentation.

Reply:
Are we leaving this as is ...? or changing it back?